### ISSUE TITLE
recognize unsuccessfully installed conda packages

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/galaxy#5039]
For multiqc I had an error during installation: `pytz-2017.3-py_2 requires a minimum conda version of 4.3.` 
The conda package was only partilally installed (in particular no multiqc binary). Except for some (difficult to spot) message in the log there was no sign in galaxy that the installation was unsuccessful. 

I have manually upgraded conda (automatic would be nice) to the version specified in https://github.com/galaxyproject/galaxy/pull/4701 then installation worked.

**Comment by mvdbeek** [comment]
xref https://github.com/galaxyproject/galaxy/issues/5024

**Comment by bernt-matthias** [comment]
Problem still persists. The ..checkenv test suggested by @bgruening  in https://github.com/galaxyproject/galaxy/issues/5024 also does not help (it returns nothing and has return code 0).

**Comment by bernt-matthias** [comment]
I also checked that the conda create command has return code 0: 

In the current case planemo test executed: 

`/home/berntm/miniconda3/bin/conda create -y --override-channels --channel iuc --channel bioconda --channel conda-forge --channel defaults --channel r --name __enasearch@0.1.1 enasearch=0.1.1`

which printed 

`Installing conda-forge::wheel-0.30.0-py_1 requires a minimum conda version of 4.3.`

return code was 0

**Comment by nsoranzo** [comment]
Returning 0 seems a conda create bug, but 4.2.x is too old, the upstream code has changed completely.

**Comment by bernt-matthias** [comment]
I agree. For the mentioned cases it is the best to just upgrade conda. 

In general the question is if conda now returns useful return codes and if galaxy checks for them.
